Classes/Issues/Comments/Reactions/IssueCommentReactionViewModel.swift

```diff
@@ -17,16 +17,33 @@ extension ReactionContent: Hashable {
 
 final class IssueCommentReactionViewModel: ListDiffable {
 
+    static let allReactions: [ReactionContent] = [
+        .thumbsUp,
+        .thumbsDown,
+        .laugh,
+        .hooray,
+        .confused,
+        .heart,
+        .rocket,
+        .eyes
+    ]
+
     let models: [ReactionViewModel]
     private let map: [ReactionContent: ReactionViewModel]
     private let flatState: String
 
     init(models: [ReactionViewModel]) {
-        self.models = models
+        let reactions = IssueCommentReactionViewModel.allReactions
+        let sortedModels = models.sorted { (m1, m2) -> Bool in
```

Comment:
nit: 
instead of `m1`, `m2`, why not `lhs`, `rhs`?
additionally for `m1Idx`, `m2Idx`, why not `lhsContent` and `rhsContent`

Reply:
`lhs` & `rhs` totally makes sense!
Not sure about `xContent` as it's an index of content in predefined array for order.

Reply:
@viktorasl what is the status of this? 😊

Reply:
Oh sorry, I thought I've fixed this. On it.

Reply:
So sorry it took so long. Fixed now.